tools/shared.py

```diff
@@ -3408,6 +3410,8 @@ def make_fetch_worker(source_file, output_file):
 V8_ENGINE = fix_js_engine(V8_ENGINE, listify(V8_ENGINE))
 JS_ENGINES = [listify(engine) for engine in JS_ENGINES]
 WASM_ENGINES = [listify(engine) for engine in WASM_ENGINES]
+if not CACHE:
+  CACHE = os.path.expanduser(os.path.join('~', '.emscripten_cache'))
```

Comment:
should we put a default value in `tools/settings_template.py`?

Reply:
No.. we don't normal users to be setting this.  Its mostly for the benefit of emsdk.  See https://github.com/emscripten-core/emsdk/pull/495.

Right now emsdk has to use the environment variable which I'd like to change.